app.js

```diff
@@ -0,0 +1,12 @@
+const express = require("express");
+const app = express();
+
+const { getTopics } = require("./controllers/controllers.js");
+
+app.get("/api/topics", getTopics);
+
+app.use((err, req, res, next) => {
+  console.log(err);
```

Comment:
Have a think about how to handle:
1. What if the user inputs an endpoint that does not exist/ we do not cater to (plus test).
2. What if there's some sort of error beyond our control? (no need to test but does need to be handled, please!)